src/openrct2/EditorObjectSelectionSession.cpp

```diff
@@ -38,7 +38,7 @@ std::vector<uint8_t> _objectSelectionFlags;
 int32_t _numSelectedObjectsForType[EnumValue(ObjectType::Count)];
 static int32_t _numAvailableObjectsForType[EnumValue(ObjectType::Count)];
 
-static void setup_in_use_selection_flags();
+static void setup_in_use_selection_flags(bool onlyUsed);
```

Comment:
These changes are unrelated remove this commit.

Reply:
What the.. how did that even get in there?! Goddamn git screwing up again. That is part of an entirely different pull request, I don't know why that's here.